features/fixtures/event_protocol_formatter/failed.ndjson

```diff
@@ -1,4 +1,4 @@
-{"type":"source","uri":"features/a.feature","data":"Feature: a feature\n  Scenario: a scenario\n    Given a step","media":{"encoding":"utf-8","type":"text/vnd.cucumber.gherkin+plain"}}
+{"type":"source","uri":"features/a.feature","data":"Feature: a feature\n  Scenario: a scenario\n    Given a step","media":{"encoding":"utf-8","type":"text/x.cucumber.gherkin+plain"}}
```

Comment:
@aslakhellesoy do you think this should be considered a breaking change?

Reply:
In theory, yes. In practice, no, as nobody (to my knowledge) is consuming these events yet.

No need for a major bump.